src/hash.jl

```diff
@@ -99,6 +99,7 @@ begin
     hash_idx += 1
   end
 end
+__init__()
```

Comment:
Should the `__init__()` call be stored inside a `if VERSION <= v"0.3"` or something?